client/state/sites/reducer.js

```diff
@@ -40,6 +41,14 @@ const VALID_SITE_KEYS = Object.keys( sitesSchema.patternProperties[ '^\\d+$' ].p
  */
 export function items( state = {}, action ) {
 	switch ( action.type ) {
+		case WORDADS_SITE_APPROVE_REQUEST_SUCCESS:
```

Comment:
Why are we storing state on the site subtree, when we've also added a root `state/wordads` tree?